### ISSUE TITLE
Output not provided when no matching commit is found

### ISSUE THREAD

**Issue by adamkudrna** [EndBug/version-check#30]
**Describe the bug**

I think an output should be always provided, not only when there is a commit with matching version number in the commit message and in the diff.

The `README` says:

> `changed`: either "true" or "false"

The example with `steps.check.outputs.changed != 'true'` could become `steps.check.outputs.changed == 'false'`. The `!= 'true'` comparison is currently triggered because there is another, non-boolean output which is logged as an empty string `''` and also complies with `!= 'true'`.

**To Reproduce**

1. Push a commit without changes in `package.json`.
2. Get this workflow triggered:

```yml
      - name: Check package.json for version change
        id: check_package_version
        uses: EndBug/version-check@v1.6.0

      - name: Log when version changed
        if: steps.check_package_version.outputs.changed == 'true'
        run: 'echo "Version change found in commit ${{ steps.check_package_version.outputs.commit }}! New version: ${{ steps.check_package_version.outputs.version }} (${{ steps.check_package_version.outputs.type }})"'

      - name: Log when version unchanged
        if: steps.check_package_version.outputs.changed != 'true'
        run: 'echo "No version change (${{ steps.check_package_version.outputs.changed }})"'
```

(Source: https://github.com/react-ui-org/react-ui/commit/14bd0bce71031da78d133fcfc9e31d20a854135b#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R60)

**Expected behavior**

`steps.check_package_version.outputs.changed` outputs `'false'`.

**Actual behavior**

`steps.check_package_version.outputs.changed` outputs `''`.

**Screenshots**

<img width="458" alt="Snímek obrazovky 2020-11-11 v 13 15 03" src="https://user-images.githubusercontent.com/5614085/98810843-17b19700-2420-11eb-9451-f72ea0edae5e.png">

**Comment by EndBug** [comment]
Hi, thanks for the issue. I agree that it's better to always give an output, I'll change it in the next version. SInce this could potentially be a breaking change, I'll release it in the next major version 👍

**Comment by EndBug** [comment]
@all-contributors please add @adamkudrna for idea and docs

**Comment by EndBug** [comment]
Resolved by release [`v2.0.0`](https://github.com/EndBug/version-check/releases/tag/v2.0.0) (you can also use `v2` and `latest`)

**Comment by adamkudrna** [comment]
OK, thanks!
